python/samples/04.ai.f.dataSource.azureOpenAI/sample.env

```diff
@@ -0,0 +1,4 @@
+# This is an example file of how to set up environment variables. You can duplicate this file and add the appropriate keys.
+OPENAI_KEY=
```

Comment:
could you remove requirement of OPENAI_KEY here?

Reply:
Removed.